test-app/frontend/src/forms/form/Form.test.tsx

```diff
@@ -0,0 +1,8 @@
+// TODO-rtotale: Add tests
+describe('FormCard component', () => {
+  it('should show default view', () => {
+    console.log('Test');
+  });
+});
+
+export {};
```

Comment:
Why the export? Oh wait, cuz no imports yet right?

Reply:
Yeah I will add the tests once I fix the heap size issue and the rest too actually